lib/galaxy/datatypes/proteomics.py

```diff
@@ -72,7 +72,7 @@ def display_peek(self, dataset):
 class ProtXmlReport(Tabular):
     """protxml converted to tabular report"""
     edam_data = "data_2536"
-    file_ext = "tsv"
+    file_ext = "protxml.tsv"
```

Comment:
@jmchilton The `file_ext` attribute is not used when downloading a history dataset, in fact if you have a "protxml.tsv" dataset on Galaxy Main and click the disk icon, the downloaded file will have ".protxml.tsv" extension. It is instead used by the `guess_ext` function in https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/datatypes/sniff.py#L369 .

This PR fixes a problem for the upload of hmm2 and hmm3 files, if you upload a text file starting with `HMMER2.0` on Galaxy Main, the sniffer recognizes it as `hmm` instead of `hmm2` because the `file_ext` attribute was wrong.